index.html

```diff
@@ -63,7 +66,7 @@
           <i class="fas fa-search"></i>
```

Comment:
- [ ] Your search form field should use the form tag 👇🏾 
![Screen Shot 2021-03-01 at 7 28 13 PM](https://user-images.githubusercontent.com/48804258/109541585-43868480-7ac4-11eb-9584-29a0070e9ce2.png)